### PR TITLE
TUA-116 add club pop up can be opened in different ways

### LINKED ISSUE #214: TUA-116  'Додати гурток' pop-up can be opened in different ways

### PR DESCRIPTION
**Pull request by Natalia62**

### CODE REVIEW

File: src/main/java/com/academy/ui/pages/ProfilePage.java [hunk 1]
```diff
@@ -74,6 +74,12 @@ public void centersDropDownClick() {
         myCentersDropDown.click();
     }
 
+    public AddClubPopUpComponent openAddClubPopUp() {
+        System.out.println(addButtonClick().get(0).getText());
```
Comment (OksankaV) [inline src/main/java/com/academy/ui/pages/ProfilePage.java]: probably, you forgot to remove this)

File: src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerProfilePage.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        AddClubPopUpComponent addClubPopUpComponent= homePage.header.addClubButtonClick();
+        addClubPopUpComponent.waitPopUpOpen(10);
+        softAssert.assertTrue(addClubPopUpComponent.isOpen(),
+                "PopUp not opened clicking the button 'Add club' in HomePage");
+
+        addClubPopUpComponent.getCloseButton().click();
+
+        ProfilePage profilePage = homePage.header.openUserMenu().clickProfile();
+        AddClubPopUpComponent addClubPopUp = profilePage.openAddClubPopUp();
+        System.out.println(addClubPopUp.getWebElement().getText());
```
Comment (OksankaV) [inline src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerProfilePage.java]: and this

**Review by OksankaV — APPROVED** [review]

File: src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerProfilePage.java (hunk 2 above)
Comment (Natalia62) [reply src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerProfilePage.java]: Yes, thanks :)

File: src/main/java/com/academy/ui/pages/ProfilePage.java (hunk 1 above)
Comment (Natalia62) [reply src/main/java/com/academy/ui/pages/ProfilePage.java]: Ooops... Removed

**Review by OksankaV — APPROVED** [review]

**Pull request merged**